### DISCUSSION TITLE
Exclude certain polyfill from core-js

### DISCUSSION THREAD

**Discussion by olegmanby** [babel/babel#13998] [General]
Hi I faced with issue - typed array polyfill of uint8array works very slow in ie10. I would like to get rid of it.
How can I do it?

**Comment by nicolo-ribaudo** [comment]
You can inject `core-js` polyfills using `babel-plugin-polyflls-corejs3` with the [`exclude`](https://github.com/babel/babel-polyfills/blob/main/docs/usage.md#include-exclude) option.

**Reply by olegmanby** [reply]
Many thanks!

**Comment by olegmanby** [comment]
@nicolo-ribaudo 

`[
        "polyfill-corejs3", 
        {
          "method": "usage-global",
          "exclude": ["es.typed-array.uint8-array"]
        }
      ],`

It doesn't work, what's wrong

**Reply by nicolo-ribaudo** [reply]
Can you share your full config, your input code, and what you mean by "it does't work"?

**Comment by olegmanby** [comment]
@nicolo-ribaudo  full config 

```
return {
    "presets": [
      "@babel/preset-env",
      "@babel/preset-react",
      "@babel/preset-flow"
    ],
    "plugins": [
      [
        "polyfill-corejs3",
        {
          "ignoreBrowserslistConfig": true,
          "method": "entry-global",
          "exclude": ["es.typed-array.uint8-array"],
          "targets": [
            "> 0.5%",
            "last 3 versions",
            "not ie < 11"
          ],
        }
      ],
      [
        "@babel/plugin-transform-runtime", 
        {
          "absoluteRuntime": false,
          "corejs": false,
          "version": "7.16.3"
        }
      ],
      "@babel/plugin-proposal-export-default-from",
      "lodash"
    ]
  }
```
input `import "core-js/stable";`

I expect that ie10 will be using native uint8array, but it still using polyfill from core-js
